.github/workflows/linux_wheel.yml

```diff
@@ -0,0 +1,69 @@
+name: Build Linux Wheels
+
+on:
+  pull_request:
+
+jobs:
+  generate-build-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    container:
+      image: python:3.9
+    steps:
+      - name: Clone pytorch/pytorch
+        uses: actions/checkout@v2
+      - id: set-matrix
+        run: |
+          # outputting for debugging purposes
```

Comment:
[apparently](https://github.com/pytorch/pytorch/runs/1745638357) omitting the name here causes GHA to use the first line of the shell script as the step name